lib/scoring/score.js

```diff
@@ -90,7 +90,9 @@ function scorePopularity(popularity, aggregation) {
     const scores = {
         communityInterest: calculateScore(popularity.communityInterest, aggregation.communityInterest),
         downloadsCount: calculateScore(popularity.downloadsCount, aggregation.downloadsCount),
-        downloadsAcceleration: calculateScore(popularity.downloadsAcceleration, aggregation.downloadsAcceleration),
+        downloadsAcceleration: popularity.downloadsAcceleration > 0 ?
```

Comment:
Shouldn't `calculateScore` handles this inside in case other properties with the same behavior arise?